eks_fargate/README.md

```diff
@@ -104,6 +104,176 @@ metadata:
+           --from-literal api-key=<YOUR_DATADOG_API_KEY> --from-literal token=<CLUSTER_AGENT_ TOKEN>
+   kubectl create secret generic datadog-secret -n fargate \
+           --from-literal api-key=<YOUR_DATADOG_API_KEY> --from-literal token=<CLUSTER_AGENT_TOKEN>
+   ```
+   For more information how these secrets are used, see the [Cluster Agent Setup][35].
+
+###### Setup
+
+1. Install the Datadog Agent with the Cluster Agent and Admission Controller enabled:
+
+   ```sh
+   helm install datadog datadog/datadog -n datadog-agent \
+       --set datadog.clusterName=cluster-name \
+       --set agents.enabled=false \
+       --set datadog.apiKeyExistingSecret=datadog-secret \
+       --set clusterAgent.tokenExistingSecret=datadog-secret \
+       --set clusterAgent.admissionController.agentSidecarInjection.enabled=true \
+       --set clusterAgent.admissionController.agentSidecarInjection.provider=fargate
+   ```
+   **Note**: Use `agents.enabled=false` for a Fargate-only cluster. On a mixed cluster, set `agents.enabled=true` to create a DaemonSet for monitoring workloads on EC2 instances.
+
+2. After the Cluster Agent reaches a running state and registers Admission Controller mutating webhooks, an Agent sidecar is automatically injected into any pod created with the label `agent.datadoghq.com/sidecar:fargate`. 
+   **The Admission Controller does not mutate pods that are already created**.
+
+**Example result**
+
+The following is a `spec.containers` snippet from a Redis deployment where the Admission Controller injected an Agent sidecar. The sidecar is automatically configured using internal defaults, with additional settings to run in an EKS Fargate environment. The sidecar uses the image repository and tags set in the Helm values. Communication between Cluster Agent and sidecars is enabled by default. 
+
+{{< highlight yaml "hl_lines=7-29" >}}
+  containers:
+  - args:
+    - redis-server
+    image: redis:latest
+  # ...
+  - env:
+    - name: DD_API_KEY
+      valueFrom:
+        secretKeyRef:
+          key: api-key
+          name: datadog-secret
+    - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: token
+          name: datadog-secret
+    - name: DD_EKS_FARGATE
+      value: "true"
+    # ...
+    image: gcr.io/datadoghq/agent:7.51.0
+    imagePullPolicy: IfNotPresent
+    name: datadog-agent-injected
+    resources:
+      limits:
+        cpu: 200m
+        memory: 256Mi
+      requests:
+        cpu: 200m
+        memory: 256Mi
+{{< /highlight >}}
+
+###### Sidecar profiles and custom selectors
+
+To further configure the Agent or its container resources, use the Helm property `clusterAgent.admissionController.agentSidecarInjection.profiles` to add environment variable definitions and resource settings. With this property, you can configure a custom selector to target workload pods instead of updating the workload to add `agent.datadoghq.com/sidecar:fargate` labels.
```

Comment:
```suggestion
To further configure the Agent or its container resources, use the Helm property `clusterAgent.admissionController.agentSidecarInjection.profiles` to add environment variable definitions and resource settings. Using `clusterAgent.admissionController.agentSidecarInjection.selectors` property, you can configure a custom selector to target workload pods instead of updating the workload to add `agent.datadoghq.com/sidecar:fargate` labels.
```